Automattic/pocket-casts-android
# Migrate player controls to Compose

## Description
This migrates player controls to Compose.

## Testing Instructions
1. Play an episode
2. Open full screen player
3. ✅ Notice that player controls are rendered properly 
4. ✅ Notice that player controls work as intended

## Screenshots or Screencast 

![image](https://github.com/user-attachments/assets/e3497a7b-d49a-4666-9ce6-e2d34bf22db8)

Portrait Normal | Large Display
----|----
<img width=200 src="https://github.com/user-attachments/assets/52fd02fc-f2fc-4bce-ade6-4f7cb25e1c99"/>| <img width=200 src="https://github.com/user-attachments/assets/95f5b11a-7e08-451d-a331-82b079470120"/>


Landscape 

<img width=500 src="https://github.com/user-attachments/assets/98ddbcbb-53e2-452d-ad46-1d28521d6d6a"/>

Button Animations

https://github.com/user-attachments/assets/a375cc12-0027-4600-b1b3-afc101550ec8


## Checklist
 
#### I have tested any UI changes...
